### PR TITLE
AYS-193 | Added positive scenarios for application

### PR DESCRIPTION
**Pull request by Nbhtozcn**
approval and rejection endpoints.

## Pull Request

**Description:**
Provide a brief description of the changes in this pull request.

**Related Issue:**
Closes #[AYS-193]

**Changes Made:**

- Added positive scenarios for application rejection and approval.

- Removed HttpStatus from specifications and assertions.

- Deleted location and assignment-related code.

- Enabled previously failed test cases for future updates and raised corresponding Jira issues

- Applied AysResponseSpec classes to several classes.

**Additional Information:**
Add any additional information that might be relevant to reviewers.

## Checklist
- [x] I have read and followed the [contribution guidelines](README.md#contributing) for this project.
- [x] I have added tests (if applicable).
- [x] All tests are passing.
- [x] I have updated the documentation (if necessary).

## Reviewer Instructions
Provide instructions or guidelines for reviewers to follow when assessing your pull request.

[AYS-193]: https://afetyonetimsistemi.atlassian.net/browse/AYS-193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/test/java/org/ays/tests/institution/adminregistrationmanagementservice/PostAdminRegistrationApplicationRejectTest.java [hunk 1]
```diff
@@ -1,100 +1,92 @@
...
+    @Test(groups = {"Regression", "SuperAdmin", "Smoke"})
+    public void rejectApplicationPositive() {
+        String applicationId = InstitutionEndpoints.generateApplicationID();
+        InstitutionEndpoints.postRegistrationApplicationIDComplete(applicationId, RegistrationApplicationCompletePayload.generate());
+        Response response = InstitutionEndpoints.postRegistrationApplicationReject(applicationId, RejectReason.generate());
+        response.then()
+                .spec(AysResponseSpecs.expectSuccessResponseSpec());
+        Response getStatus = InstitutionEndpoints.getRegistrationApplicationId(applicationId);
+        getStatus.then()
+                .body("response.status", equalTo("REJECTED"));
```
Comment (agitrubard) [inline src/test/java/org/ays/tests/institution/adminregistrationmanagementservice/PostAdminRegistrationApplicationRejectTest.java:23]: `"REJECTED"` yerine enum objeler oluşturularak kullanılabilir

**Review by agitrubard — CHANGES_REQUESTED** [review]

**Comment by agitrubard** [comment]
@Nbhtozcn branch isimlerinde `|` yerine `/` kullanabiliriz

File: src/test/java/org/ays/payload/ApplicationStatus.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package org.ays.payload;
+
+public enum ApplicationStatus {
```
Comment (agitrubard) [inline src/test/java/org/ays/payload/ApplicationStatus.java:3]: Aşağıdaki şekilde güncellememiz daha iyi olur, ilerleyen dönemde başka başvuru aşamaları da olacak, karışabilir
```suggestion
public enum AdminUserRegisterationApplicationStatus {
```

**Review by agitrubard — CHANGES_REQUESTED** [review]

File: src/test/java/org/ays/payload/ApplicationStatus.java (hunk 2 above)
Comment (Nbhtozcn) [reply src/test/java/org/ays/payload/ApplicationStatus.java:3]: Haklisin @agitrubard. Değiştirip yolluyorum.

**Review by agitrubard — APPROVED** [review]

**Pull request merged**